websites/H/HEO Systems/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.12",
+	"apiVersion": 1,
+	"author": {
+		"id": "496673945211240462",
+		"name": "hamzieee"
+	},
+	"service": "HEO Systems",
+	"description": {
+		"en": "HEO Systems is a hosting company, mainly focused on hosting game servers.",
+		"nl": "HEO Systems is een hostingbedrijf, voornamelijk gericht op het hosten van game servers."
+	},
+	"url": [
+		"www.heo-systems.net",
+		"clients.heo-systems.net",
+		"ptero.heo-systems.net",
+		"heos-hosting.site"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.ibb.co/ThD1x82/heo-systems-modern-transparent-1500w-1.png",
```

Comment:
Logo must be 512x512

Reply:
This image is 512x512

Reply:
Ah, I noticed in the link it says "1500w", but the image is really 512x512. Maybe you thought it was 1500x1500. I'll change it